packages/adblocker-benchmarks/Makefile

```diff
@@ -4,6 +4,15 @@ all: deps run
 requests.json:
 	curl https://cdn.cliqz.com/adblocking/requests_top500.json.gz | gunzip > requests.json
 
+# VERSION: e1222d16430d754e8a2499fd8c58f49880d6c283
+./blockers/ublock:
+	git clone https://github.com/gorhill/ublock ./blockers/ublock
+	git clone https://github.com/uBlockOrigin/uAssets.git ./blockers/uAssets
+	cd ./blockers/ublock && git checkout --quiet e1222d16430d754e8a2499fd8c58f49880d6c283 && ./tools/make-nodejs.sh
+	npm install --no-save ./blockers/ublock/dist/build/uBlock0.nodejs
+
+ubo-snfe: ./blockers/ublock
```

Comment:
What does `snfe` stand for?

Reply:
It stands for "static net filtering engine."